scripts/zkai_native_seq32_attention_mlp_adapter_variant_selector_gate.py

```
@@ -0,0 +1,764 @@
+
+def source_artifacts() -> list[dict[str, Any]]:
+    return [artifact_record(artifact_id, path) for artifact_id, path in SOURCE_ARTIFACT_SPECS]
+
+
+def load_variant_rows() -> list[dict[str, Any]]:
+    accounting = _dict(read_json_file(ACCOUNTING, "variant accounting"), "variant accounting")
+    rows = _list(accounting.get("rows"), "accounting rows")
+    if len(rows) != len(EXPECTED_ROWS):
+        raise NativeSeq32AdapterVariantSelectorGateError("variant inventory drift")
+    parsed = []
+    for expected, row in zip(EXPECTED_ROWS, rows):
```

Ruff (0.15.13)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)